plugins/webapi/mana/logsHandler.go

```diff
@@ -0,0 +1,68 @@
+package mana
+
+import (
+	"net/http"
+
+	"github.com/iotaledger/goshimmer/packages/mana"
+	manaPlugin "github.com/iotaledger/goshimmer/plugins/mana"
+	"github.com/iotaledger/hive.go/identity"
+	"github.com/labstack/echo"
+	"github.com/mr-tron/base58"
+)
+
+// getEventLogsHandler handles the request.
+func getEventLogsHandler(c echo.Context) error {
```

Comment:
Would be nice to:
 - Include the time of the current oldest logged event in the response, to know that that is the "oldest" event I could expect.
 - Add a time range in the request.
 - Possibility to get all logs, not just selected nodes.
 - Return the events sorted.

Reply:
>Include the time of the current oldest logged event in the response, to know that that is the "oldest" event I could expect.

would mean that we travel the whole storage every time even if we wanted to get logs only for 1 node.
I don't think its worth it.